### PR TITLE
Add r-paco recipe

### PR DESCRIPTION
**Pull request by nick-youngblut**
My first contribution. The build is passing on my machine.

### CODE REVIEW

File: recipes/r-paco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+about:
+  home: http://www.uv.es/cophylpaco/
+  license: MIT
+  summary: Procrustes analyses to infer co-phylogenetic matching between pairs of (ultrametric)
+    phylogenetic trees.
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - nick-youngblut
```
Comment (bgruening) [inline recipes/r-paco/meta.yaml]: Can you please add:
```
    - johanneskoester
    - bgruening
    - daler
    - jdblischak
   - cbrueffer
```

here as well? was this not added by the script?

**Comment by bgruening** [comment]
@jdblischak @cbrueffer does this look good to you as well?

File: recipes/r-paco/meta.yaml (hunk 1 above)
Comment (nick-youngblut) [reply recipes/r-paco/meta.yaml]: Sorry! I read the docs wrong, and so I thought that I was supposed to be the sole maintainer.

**Comment by cbrueffer** [comment]
Looks good :+1:

**Comment by bgruening** [comment]
Thanks @nick-youngblut!

**Pull request merged**